about.html

```diff
@@ -0,0 +1,99 @@
+                    <li><i class="fa-brands fa-facebook-f"></i></li>
+                    <li><i class="fa-brands fa-twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+            </div>
+            <nav class="nav-bar">
+                <div class="logo">
+                    <a href="index.html"><img src="images/logo_square_large_print.webp" alt=""></img></a>
+                </div>
+                <ul class="nav-list">
+                  <li class="nav-item">
+                    <a  href="index.html">Home</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="about.html">About</a>
+                  </li>
+                  <li class="nav-item">
+                    <a  href="">Program</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="">Join</a>
+                  </li>
+                  <li class="nav-item">
+                    <a  href="">Sponsors</a>
+                  </li>
+                  <li class="nav-item">
+                    <a  href="#">News</a>
+                  </li>
+                  <li class="nav-item">
+                    <a href="#" class="campaign">CC campaign</a>
+                  </li>
+                </ul>
+            </nav>
+        </header>
+        <div class="about">
+            <h3 class="hello">"Hello! Sharing World!"</h3>
+            <h1 class="title">The Event Listener Music Festival 2015</h1>
+            <div class="intro-about">
+                <p>The Event Listener Music Festival brings together the community of music lovers, adventurers and bohemians who travel around the globe in search for the harmony this festival delivers. This year, we hope to expand our invitation list more individuals and groups.</p>
+            </div>
+        </div>
+        <div class="contact">
+            <p>Please contact us per Email for any further questions about The Event Listener Music Festival 2015!</p>
+            <p class="email">telmf2015@cckorea.org</p>
+        </div>
+    </section>
+    <section class="logo-section">
+        <h3 class="logo-title">The Event Listener Music Festival 2015 Logo</h3>
+        <p>The logo of TELMS 2015 was decided through the logo competition from 8.June to 7.July.</p>
+        <img src="images/logo_square_large_print.webp" alt="">
+    </section>
+    
+    <section id="previous-events">
+        <h2>See the past TELMF events </h2>
+        <p>Take a look at the last two TELM events which took place in Buenos Aires and in Warsaw</p>
+        <div class="events-desktop">
+        <div class="buenosAires">
+            <h2>2013</h2>
+            <p>TELMF 2013 in Buenos Aires</p>
+        </div>
+        <div class="warsaw">
+            <h2>2011</h2>
+            <p>TELMF 2013 in Warsaw</p>
+        </div>
+        </div>
+    </section>
+    
+    <footer id="footer-section">
```

Comment:
- [OPTIONAL] Kindly consider adding the `partners` section here on the About page on the Mobile screen size as suggested in the Behance design.
Behance design about page on mobile screen
![re](https://user-images.githubusercontent.com/96683843/234950502-2ce26df5-4018-4c44-ae23-85b7e3d03c73.jpg)